mailpoet/lib/Util/Notices/InactiveSubscribersNotice.php

```diff
@@ -32,7 +32,7 @@ public function init($shouldDisplay) {
     }
 
     // don't display notice if user has changed the default inactive time range
-    $inactiveDays = $this->settings->get('deactivate_subscriber_after_inactive_days');
+    $inactiveDays = (int)$this->settings->get('deactivate_subscriber_after_inactive_days');
```

Comment:
I'm pretty sure the inactive subscriber notice hasn't been displaying as expected for the majority of users because this wasn't being cast to an integer. The only way the conditional below would be true is if there was no `deactivate_subscriber_after_inactive_days` setting saved in the user's database, which is only true for someone who's never gone into any MailPoet settings page and clicked the save button.